base/src/main/java/uk/ac/ebi/atlas/solr/analytics/baseline/BaselineAnalyticsSearchDao.java

```diff
@@ -3,18 +3,29 @@
 import com.jayway.jsonpath.Configuration;
 import com.jayway.jsonpath.JsonPath;
 import com.jayway.jsonpath.Option;
+import org.apache.commons.math.util.MathUtils;
 import org.springframework.stereotype.Component;
-import uk.ac.ebi.atlas.profiles.baseline.BaselineExpressionLevelRounder;
 import uk.ac.ebi.atlas.search.SemanticQuery;
 import uk.ac.ebi.atlas.solr.analytics.query.AnalyticsQueryClient;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.function.DoubleUnaryOperator;
 
 @Component
 public class BaselineAnalyticsSearchDao {
-    // analyticsQueryClient.baselineFacets searches for baseline experiments
+    static final int MIN_PRECISION = 7;
```

Comment:
Why magic number 7?

Reply:
Who knows... this goes back to scientific choices by experts in gene expression. The value was chosen to accomodate proteomics and RNA-Seq experiments under the same baseline category. TPMs lowest value is 0.1 (anything below that we consider it not expressed), but proteomics experiments have [very](https://www.ebi.ac.uk/gxa/experiments/E-PROT-1) [different](https://www.ebi.ac.uk/gxa/experiments/E-PROT-5) magnitudes and measure some sort of fluorescence value IIRC.